python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -248,8 +242,11 @@ def conv2d(attrs, args):
     return True
 
 
-def qnn_conv2d(attrs, args):
+# TODO check why no annotation is here
+@tvm.ir.register_op_attr("qnn.conv2d", "target.arm_compute_lib")
```

Comment:
This shouldn't be registered. It's only supported as part of a composite pattern.

Reply:
updated